contracts/seed/Seed.sol

```diff
@@ -187,14 +204,48 @@ contract Seed {
      * @param _individualCap      The personal cap of each contributor in this class.
      * @param _price              The token price for the addresses in this clas.
      * @param _vestingDuration    The vesting duration for this contributors class.
+     * @param _classVestingStartTime The class vesting start time for the contributor class.
+     * @param _classFee           The fee for the contributor class.
      */
     function addClass(
         uint256 _classCap,
         uint256 _individualCap,
         uint256 _price,
-        uint256 _vestingDuration
+        uint256 _vestingDuration,
+        uint256 _classVestingStartTime,
+        uint256 _classFee
```

Comment:
I think we need to put a require statement for fee, let's make it always less then 45%(in initialize as well)